packages/components/src/globals/scss/_css--reset.scss

```diff
@@ -193,6 +193,11 @@
 
     * {
```

Comment:
Maybe we should switch to what is recommended in: https://css-tricks.com/inheriting-box-sizing-probably-slightly-better-best-practice/

```scss
html {
  box-sizing: border-box;
}
*, *::before, *::after {
  box-sizing: inherit;
}
```

Reply:
@joshblack I agree this would be best to do! But I"m thinking it should be a separate PR, since it would require a double-checking all components to make sure the `box-sizing` changes look good. 🤔 

I"m thinking for this PR I'm going to remove any updates to the `css--reset` and keep my changes confined to `Checkbox`